### PR TITLE
Updated 'how to read this guide' in line with latest changes

### PR DESCRIPTION
**Pull request by JayMurdoch**
- Add missing conformance sub page (declaring conformance)
- Add missing guidance sub pages (med info, variance statement)
- Add security and privacy point
- Add 'support' header page in line with other headers
- Remove duplicated comparison with other IGs point.

### CODE REVIEW

File: input/ig.xml [hunk 1]
```diff
@@ -794,15 +794,20 @@
...
+				<title value="Support"/>
+				<generation value="markdown"/>
+			</page>
+				<page>
 				<nameUrl value="downloads.html"/>
 				<title value="Downloads"/>
 				<generation value="markdown"/>
-			</page>
-			<page>
+				</page>
```
Comment (dt-r) [inline input/ig.xml:805]: This isn't a required change but be wary of changing the indenting on files - it shows up in the diff and adds to reviewer burden.

File: input/pagecontent/index.md [hunk 2]
```diff
@@ -117,18 +117,20 @@ This guide is divided into several pages which are listed at the top of each pag
   - [Medicine Information](medicine-information.html): This page provides guidance on constructing medications related resources.
   - [AUCDI](aucdi.html): This page maps the AU Core resources and elements to AUCDI data classes and data elements.
   - [Relationship with other IGs](relationship.html): This page provides guidance on the relationship between AU Core and other implementation guides.
-  - [Comparison with other national and international specifications](comparison.html): This page provides comparison between AU Core profiles and other national, or international implementation guides.
   - [AU Variance Statement](variance.html): This page documents the variance from AU Base.
   - [Comparison with other national and international IGs](comparison.html): This page provides comparison between AU Core profiles and other national, or international implementation guides.
   - [Future of AU Core](future.html): This page outlines the approach to developing AU Core and yearly update cycle.
+- [Security and Privacy](security.html): This page outlines the security and privacy obligations to consider when implementing AU Core.
```
Comment (dt-r) [inline input/pagecontent/index.md:123]: obligations is a loaded word, do not use unless you mean FHIR Obligations. These are requirements. Use the below text instead.

"This page documents the AU Core general security and privacy requirements and recommendations."

File: input/pagecontent/index.md [hunk 3]
```diff
@@ -117,18 +117,20 @@ This guide is divided into several pages which are listed at the top of each pag
...
   - [Search Parameters](search-parameters.html): This set of pages lists the search parameters extended for use in this guide for use in AU Core operations.
   - [Terminology](terminology.html): This set of pages lists the value sets and code systems defined in this guide.
   - [Capability Statements](capability-statements.html): This set of pages define the expected FHIR capabilities of AU Core Responders and Requesters.
   - [Actor Definitions](actors.html): This set of pages define the AU Core actors, AU Core Responder and AU Core Requester.
 - [Examples](examples.html): This page lists all the examples used in this guide.
-- [Downloads](downloads.html): This page provides links to downloadable artefacts that developers can use to help them implement this guide.
+- [Support](support.html): These pages provide supporting material for implementation or AU Core.    
+  - [Downloads](downloads.html): This page provides links to downloadable artefacts that developers can use to help them implement this guide.
```
Comment (dt-r) [inline input/pagecontent/index.md:132]: This text describing the downloads page is different across all three specs. There is an open PR on AU Base that is not consistent with AU Core.

Advise on the text that will be applied and make both PRs consistent; also create the AU eRequesting PR and submit. Approval of all three spec PRs will occur in parallel.

File: input/pagecontent/index.md [hunk 4]
```diff
@@ -117,18 +117,20 @@ This guide is divided into several pages which are listed at the top of each pag
...
+  - [License and Legal](license.html): This page outlines license and legal considerations relating to FHIR Implementation Guides.
```
Comment (dt-r) [inline input/pagecontent/index.md:133]: Not considerations relating to all FHIR IGs that exist. These are the licensing and legal requirements for material in AU Core. Use the text following text:
This page outlines the license and legal requirements for material in AU Core.

Apply that style to all three IGs.

File: input/pagecontent/index.md [hunk 5]
```diff
@@ -117,18 +117,20 @@ This guide is divided into several pages which are listed at the top of each pag
...
   - [Search Parameters](search-parameters.html): This set of pages lists the search parameters extended for use in this guide for use in AU Core operations.
   - [Terminology](terminology.html): This set of pages lists the value sets and code systems defined in this guide.
   - [Capability Statements](capability-statements.html): This set of pages define the expected FHIR capabilities of AU Core Responders and Requesters.
   - [Actor Definitions](actors.html): This set of pages define the AU Core actors, AU Core Responder and AU Core Requester.
 - [Examples](examples.html): This page lists all the examples used in this guide.
-- [Downloads](downloads.html): This page provides links to downloadable artefacts that developers can use to help them implement this guide.
+- [Support](support.html): These pages provide supporting material for implementation or AU Core.
```
Comment (dt-r) [inline input/pagecontent/index.md:131]: Spelling mistake "or AU Core" to be "of AU Core"

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments.

File: input/ig.xml (hunk 1 above)
Comment (JayMurdoch) [reply input/ig.xml:805]: Noted, will not amend in future.

File: input/pagecontent/index.md (hunk 2 above)
Comment (JayMurdoch) [reply input/pagecontent/index.md:123]: Fixed, thanks.

File: input/pagecontent/index.md (hunk 3 above)
Comment (JayMurdoch) [reply input/pagecontent/index.md:132]: Changed all to: "This page provides links to downloadable artefacts."

File: input/pagecontent/index.md (hunk 4 above)
Comment (JayMurdoch) [reply input/pagecontent/index.md:133]: Applied suggested sentence style to all three IGs.

File: input/pagecontent/index.md (hunk 5 above)
Comment (JayMurdoch) [reply input/pagecontent/index.md:131]: Fixed, thanks.

**Comment by JayMurdoch** [comment]
Changes applied @dt-r - please review with https://github.com/hl7au/au-fhir-base/pull/854 & https://github.com/hl7au/au-fhir-erequesting/pull/40

File: input/pagecontent/index.md [hunk 6]
```diff
@@ -125,8 +125,8 @@ This guide is divided into several pages which are listed at the top of each pag
   - [Profiles and Extensions](profiles-and-extensions.html): This page describes the profiles and extensions that are defined in this guide to exchange quality data. Each profile page includes a narrative description and guidance, formal definition and a "Quick Start" guide which summarises the supported search transactions for each profile. Although the guidance typically focuses on the profiled elements, it may also may focus on un-profiled elements to aid with implementation.
```
Comment (dt-r) [inline input/pagecontent/index.md:125]: For AUCDI line replace with:
This page maps AUCDI data groups and elements to FHIR artefacts in AU Core.

Comment (dt-r) [inline input/pagecontent/index.md:125]: remove the word "quality" from "exchange quality"

File: input/pagecontent/index.md [hunk 7]
```diff
@@ -125,8 +125,8 @@ This guide is divided into several pages which are listed at the top of each pag
...
   - [Search Parameters](search-parameters.html): This page lists the search parameters extended for use in this guide for use in AU Core operations.
   - [Terminology](terminology.html): This page lists the value sets and code systems defined in this guide.
```
Comment (dt-r) [inline input/pagecontent/index.md:127]: In terminology line replace "defined" with "supported"

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments.

File: input/pagecontent/index.md (hunk 6 above)
Comment (JayMurdoch) [reply input/pagecontent/index.md:125]: Fixed, thanks.

Comment (JayMurdoch) [reply input/pagecontent/index.md:125]: Removed, thanks.

File: input/pagecontent/index.md (hunk 7 above)
Comment (JayMurdoch) [reply input/pagecontent/index.md:127]: Fixed, thanks.

File: input/pagecontent/index.md (hunk 6 above)
Comment (dt-r) [reply input/pagecontent/index.md]: 1. Fixed
2. Can't see the change.

Comment (dt-r) [reply input/pagecontent/index.md]: Can't see this change.

**Review by dt-r — CHANGES_REQUESTED** [review]
Looks like a commit is missing. See unresolved comments.

File: input/pagecontent/index.md (hunk 6 above)
Comment (JayMurdoch) [reply input/pagecontent/index.md]: Sorry, fixed.

File: input/pagecontent/index.md [hunk 8]
```diff
@@ -115,14 +115,14 @@ This guide is divided into several pages which are listed at the top of each pag
 - [Guidance](guidance.html): These pages list the guidance for this guide.
   - [General Guidance](general-guidance.html): This page provides guidance on using the profiles defined in this guide.
   - [Medicine Information](medicine-information.html): This page provides guidance on constructing medications related resources.
-  - [AU Core Data for Interoperability](aucdi.html): This page maps AUCDI data groups and elements to FHIR artefacts in AU Core.
+  - [AU Core Data for Interoperability](aucdi.html): This page provides guidance on the relationship between AU Core, AUCDI, and other implementation guides.
```
Comment (dt-r) [inline input/pagecontent/index.md:118]: This change, was applied to the wrong line. Put back the line for AU Core Data for Interoperability and move this new text to the line for Relationship with Other IGs.

File: input/pagecontent/index.md (hunk 6 above)
Comment (dt-r) [reply input/pagecontent/index.md]: This change for number 2, overwrote the change to number 1 and has been was applied to the wrong line. Put back the line for AU Core Data for Interoperability and move this new text to the line for Relationship with Other IGs.

**Review by dt-r — CHANGES_REQUESTED** [review]
One unresolved conversation to address.

File: input/pagecontent/index.md (hunk 6 above)
Comment (JayMurdoch) [reply input/pagecontent/index.md]: Fixed, sorry.

**Review by dt-r — APPROVED** [review]

**Pull request merged**